readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -129,8 +129,9 @@ def append_conf(self, **kwargs):
         # Print the contents of conf.py in order to make the rendered
         # configfile visible in the build logs
         self.run(
-            'cat', os.path.basename(outfile_path),
-            cwd=os.path.dirname(outfile_path),
+            'cat', os.path.relpath(outfile_path,
+                                   project.checkout_path(self.version)),
+            cwd=project.checkout_path(self.version),
```

Comment:
Yea it should use the slug, `self.version` is an object. Looks good to me otherwise. 

I've long wanted to have a full listing of configuration for the build (perhaps a "generated YAML config", if we ever switch over to a full YAML build setup) so that users find it easier to debug the settings that were used for that specific build.